File: python/counting_duplicates/duplicate_count.py
def duplicate_count(text):
    total = 0
    text = text.lower()
    unq = set(text)

    if len(text) == 0 or len(unq) == len(text):
        return total

    for i in unq:
        if text.count(i) > 1:
            total += 1

    return total

File: javascript/counting_duplicates/duplicate_count.js
function duplicateCount(text) {
  let total = 0
  text = text.toLowerCase()
  let unq = new Set(text)

  if (text.length === 0 || text.length === unq.size) return total

  for (let i of unq) {
    if (text.match(new RegExp(`${i}`, "g")).length > 1) {
      total += 1
    }
  }

  return total
}
module.exports = duplicateCount;
